applications/FemToDemApplication/python_scripts/FemDemMechanicalSolver.py

```diff
@@ -330,7 +331,7 @@ def _execute_after_reading(self):
             params.AddValue("bodies_list",self.settings["bodies_list"])
 
         # CheckAndPrepareModelProcess creates the computating_model_part
-        import check_and_prepare_model_process
+        import KratosMultiphysics.FemToDemApplication.check_and_prepare_model_process as check_and_prepare_model_process

```

Comment:
the imports have to be at the top of the file unless not possible 
according to Pooyan and Charlie we otherwise get problems with recursive imports in some configurations

Reply:
It is also the good practice according to the PEP 8 style guide.

Reply:
okkey! Changing it :D